frontend/src/metabase/core/components/FormCategoryInput/CategoryFieldInput/CategoryFieldInput.tsx

```diff
@@ -31,22 +31,22 @@ const DefaultTokenFieldLayout = ({
 
 interface CategoryFieldInputProps {
   value: string | number;
-  field: Field;
+  fieldInstance: Field;
   onChange: (newValue: string) => void;
 }
 
 function CategoryFieldInput({
   value,
-  field,
+  fieldInstance,
   onChange,
 }: CategoryFieldInputProps) {
   return (
     <FieldValuesWidgetContainer>
       <StyledFieldValuesWidget
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-ignore, typescript is very confused about the value type for this non-tsx component
-        value={value ? [String(value)] : ([] as any)}
-        fields={[field]}
+        value={value ? [String(value)] : ([""] as any)}
```

Comment:
This expression feels a bit too complicated. Is there anything we can do to make it easier to read?

Reply:
🤦  yes. 😳 

```
value={[String(value ?? '')]}
```